airflow/api_fastapi/db.py

```diff
@@ -19,6 +19,9 @@
 
 from typing import TYPE_CHECKING
 
+from sqlalchemy.sql import Select
+
+from airflow.api_fastapi.parameters import BaseParam
```

Comment:
Should move these into the `if TYPE_CHECKING` block below, no?

Reply:
Would not `ruff` tell you so :-D

Reply:
"TCH001" and "TCH002" are disabled on `api_fastapi` folder. Same reason as for `serialization/pydantic/` pydantic manipulates annotations at runtime and this things will fail.

Also `Annotated` for old python version and future __annotations__ are not working well.